src/endpoints/test/validate.http

```diff
@@ -26,4 +26,12 @@ content-type: application/json
 
 {
     
+}
+#####
+GET http://localhost:5000/api/mint/getMagicLinkFromAirtable
+content-type: application/json
+id-token: eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IlRZT2dnXy01RU9FYmxhWS1WVlJZcVZhREFncHRuZktWNDUzNU1aUEMwdzAifQ.eyJpYXQiOjE2ODgxMzUyOTUsImF1ZCI6IkJFc3NHT0ZaRFVTT1BwelhadmRTUTAyblhCUVVDQTEzRVE3cElJZFBmbmljaEc1NUpSOGtVMHBVNXpSR28teEp1emEwU1VsNUd4OWUyUFhPdXZNejFvZyIsIm5vbmNlIjoiMDIzYzNmZTk5OTA5ZjNhMjhhYWJmM2E0MWIyYTc5MzY0YTNlNTdlYmI4MThjOTYyNDMyZjA4MTIyNTZhNDRmN2E1IiwiaXNzIjoiaHR0cHM6Ly9hcGkub3BlbmxvZ2luLmNvbSIsIndhbGxldHMiOlt7InB1YmxpY19rZXkiOiJiYzQ4YTIzMmRkMjQwYjExZTgxNGJlNWU3MTA1NDMyN2I0ODNlYTQ5YTQyMzQ3ZjY3NDU0ZGNmZDFiNDM3ODJiIiwidHlwZSI6IndlYjNhdXRoX2FwcF9rZXkiLCJjdXJ2ZSI6ImVkMjU1MTkifV0sImVtYWlsIjoibWVoZGlkZWhkYXI4OUBnbWFpbC5jb20iLCJuYW1lIjoibWVoZGlkZWhkYXI4OUBnbWFpbC5jb20iLCJ2ZXJpZmllciI6InRvcnVzIiwidmVyaWZpZXJJZCI6Im1laGRpZGVoZGFyODlAZ21haWwuY29tIiwiYWdncmVnYXRlVmVyaWZpZXIiOiJ0a2V5LWF1dGgwLWVtYWlsLXBhc3N3b3JkbGVzcy1scmMiLCJleHAiOjE2ODgyMjE2OTV9.lKR-J_vEeFKYer9NM7-NTj2yzBOOuqOoVf8mgyvrABkZxOqa8RYRsDomcvIx8CtWbfgfp-EZvSbyqnjoPF2VmQ
```

Comment:
which ID token is that? this is a small concern since these tokens expire quickly but we gotta make sure we are not pushing these to repo